packages/demo/package.json

```diff
@@ -54,7 +55,8 @@
     "@web3-onboard/uauth": "^2.0.4",
     "@web3-onboard/zeal": "^2.0.3",
     "@web3-onboard/cede-store": "^2.0.1",
-    "vconsole": "^3.9.5"
+    "vconsole": "^3.9.5",
+    "@dapperlabs/dappauth": "^2.0.5"
```

Comment:
What is this dep needed for?
If required for Blocto can it be added to the package?

Reply:
This tool can verify signature, [see more](https://docs.blocto.app/blocto-sdk/javascript-sdk/evm-sdk/sign-message#verify-signature)